mtp_api/apps/security/serializers.py

```diff
@@ -219,19 +219,32 @@ def update(self, instance, validated_data):
 
 class CheckSerializer(serializers.ModelSerializer):
     credit = SecurityCreditSerializer()
+    full_name = serializers.SerializerMethodField('get_full_name_from_user')
 
     class Meta:
         model = Check
         fields = (
             'id',
             'credit',
+            'created',
             'status',
             'description',
             'rules',
             'actioned_at',
             'actioned_by',
+            'rejection_reason',
+            'full_name',
         )
 
+    def get_full_name_from_user(self, check):
+        if check.actioned_by is not None:
+            first_name = check.actioned_by.first_name
```

Comment:
the user object already has a `get_full_name` method. also, i think it might be better to say whose full name this is. maybe `actioned_by_name` or something?